lib/movie_app_bar.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class MovieAppBar extends StatelessWidget implements PreferredSizeWidget {
+  final String movieTitle;
+  static const Icon appBarBackIcon = Icon(Icons.arrow_back);
```

Comment:
This icon is not necessary to add it as a const, you can use it directly on line 23

Reply:
fixed